travel/config_reader.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
Looks like config parsed is another task. It should be merged into separate branch.